client/src/components/questioncard.js

```diff
@@ -1,14 +1,32 @@
+import { useState } from "react";
+
 const QuestionCard = (props) => {
-    
-    return (
-      <div className={"question-section"}>
-        <div className='question-text'>{props.question.question}</div>
-        <div className='answer-section'>
-		<button>True</button>
-		<button>False</button>
-        </div>
-      </div>
-    );
+  const [answer, setAnswer] = useState(false);
+ 
+  let handleClick = (event) => {
+    setAnswer(true);
+
+    let selectedAnswer = event.target.dataset.value; // data-value in line 15 is a custom attribute and better strategy to use than ID (since we can't have individual IDs in our example)
+    if (props.question.correct_answer === selectedAnswer) {
+      event.target.style.backgroundColor = "green"; 
```

Comment:
🔴 This isn't the correct way to style things in React. If this component re-renders, these styles could be lost.

Instead, you should store the selected answer in state with `setAnswer(selectedAnswer)` and then you would style each button like this:

```jsx
<button style={answer === props.question.correct_answer ? { backgroundColor: 'green' } : {}}>
```